cmd/worker/workercmd/cmd.go

```diff
@@ -96,10 +89,15 @@ func (m *Main) Run(ctx context.Context) error {
 		temporalsdk_activity.RegisterOptions{Name: activities.TransformSIPName},
 	)
 	w.RegisterActivityWithOptions(
-		bagit.NewCreateBagActivity(m.cfg.Bagit).Execute,
-		temporalsdk_activity.RegisterOptions{Name: bagit.CreateBagActivityName},
+		bagcreate.New(m.cfg.Bagit).Execute,
+		temporalsdk_activity.RegisterOptions{Name: bagcreate.Name},
 	)
 
+	// AIS workflow.
+	if err := ais.RegisterWorkflow(ctx, w, m.cfg.AIS); err != nil {
+		return fmt.Errorf("AIS: %w", err)
+	}
+
```

Comment:
I decided to do it this way to keep AIS integration things together as much as possible, to ease its extraction to a different worker if we need to.